config/v1/types_infrastructure.go

```diff
@@ -1343,8 +1343,9 @@ type VSpherePlatformSpec struct {
 	// ---
 	// + If VCenters is not defined use the existing cloud-config configmap defined
 	// + in openshift-config.
-	// +kubebuilder:validation:MaxItems=1
 	// +kubebuilder:validation:MinItems=0
+	// +openshift:validation:FeatureGateAwareMaxItems:featureGate="",maxItems=1
+	// +openshift:validation:FeatureGateAwareMaxItems:featureGate=VSphereMultiVCenters,maxItems=3
```

Comment:
@vr4manta just curious why only three? Is there a documented maximum?

Reply:
Yes, is in the epic.  Not sure the reasoning for 3 being the max, but for now, that's what we have it being set to.